packages/react-urql/examples/1-getting-started/src/service-worker.ts

```diff
@@ -0,0 +1,34 @@
+import { execute } from 'graphql';
+import { rootValue, schema } from './graphql';
+import gql from 'graphql-tag';
+
+self.addEventListener('install', (event: any) => {
+  console.log('GraphQL service worker installed.');
+  event.waitUntil((self as any).skipWaiting());
```

Comment:
You should be able to call `skipWaiting` without `waitUntil` afaik

Reply:
Was trying to get immediate access to network interceptions [using this example](https://serviceworke.rs/immediate-claim_service-worker_doc.html)